### DISCUSSION TITLE
Welcome to samm Discussions!

### DISCUSSION THREAD

**Discussion by bingogome** [bingogome/samm#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by slicer3d** [comment]
Hi,

I would like to know if anyone could help me with installation of SAM on my mac for segmentation using slicer3d. I would like to use this tool for segmentation of anatomy on 2d images. I attempted installation following the installation guide, but ran into a few issues. 

Thanks in advance!
